### DISCUSSION TITLE
Website broke on mobile Firefox

### DISCUSSION THREAD

**Discussion by Noratrieb** [community/community#73151] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As of a few days ago, the website is broken on mobile. When opening an issue or pull request, it only fills half the screen. Sometimes it fixes itself after a few seconds and sometimes it doesnt. When scrolling down and up again, it sometimes hides even more as shown in the screenshot. 
I am on Android Firefox. On Android Chrome, the same behavior happens, although it fixes itself more reliably after a second there.
On Firefox, I can make the problem appear by zooming out, on Chrome zooming out doesn't work.

This is a recent regression, it worked perfectly fine on Firefox a week ago.

![Screenshot_20231101-154226](https://github.com/community/community/assets/48135649/ecf99a2d-6467-491a-8b73-085025fab93b)
![Screenshot_20231101-154643](https://github.com/community/community/assets/48135649/83f3e253-c1ca-49f9-8ecb-9d895a6f983c)

**Comment by haltcase** [comment]
This actually occurs for me on every mobile (Android) browser I have installed: Samsung Internet, Firefox, and Google Chrome. It's a super annoying issue and I'm surprised it's been out there this long (~2.5 months at least) and this is the only place I see someone talking about it.

However, I've also found that it only occurs when I'm signed in, so I assume it has something to do with user state or components that break the layout. If I scroll down, all the way to the comment box, the layout fixes itself until refresh/navigation.

_edit_: seems likely to be the signed-in sidebar on the right that opens when clicking the user avatar. Opening that by clicking the avatar causes the layout to shift and fix itself.

### Samsung Internet

![Screenshot_20231228_230729_Samsung Internet.png](https://github.com/community/community/assets/16605186/ece4428c-3094-455c-81e5-c736f7d9dc89)

### Firefox

![Screenshot_20231228_230944_Firefox.png](https://github.com/community/community/assets/16605186/999eb051-4239-4d45-be7c-cf447f252df1)

### Google Chrome

![Screenshot_20231228_231145_Chrome.png](https://github.com/community/community/assets/16605186/7005bfe5-a2fd-4530-96af-64fcdc3e5f94)
